src/lib.rs

```diff
@@ -62,3 +72,40 @@ fn dotenv_files() -> Result<Vec<DirEntry>, Error> {
 
     Ok(files)
 }
+
+pub fn extract_key(input: &str) -> String {
```

Comment:
I've decided to put it here because it will be used in other checks to (i.e. unordered keys)